src/fallback.ts

```diff
@@ -233,17 +262,30 @@ export class GrpcClient {
       throw new Error('No authentication was provided');
     }
     const authHeader = await this.authClient.getRequestHeaders();
-    const serviceStub = service.create(serviceClientImpl, false, false);
+    const serviceStub = (service.create(
+      serviceClientImpl,
+      false,
+      false
+    ) as unknown) as FallbackServiceStub;
     const methods = this.getServiceMethods(service);
 
-    const newServiceStub = service.create(serviceClientImpl, false, false);
+    const newServiceStub = (service.create(
+      serviceClientImpl,
+      false,
+      false
+    ) as unknown) as FallbackServiceStub;
     for (const methodName of methods) {
-      newServiceStub[methodName] = (req, options, metadata, callback) => {
+      newServiceStub[methodName] = (
+        req: {},
+        options: {[name: string]: string},
+        metadata: {},
+        callback: Function
+      ) => {
         const [method, requestData, serviceCallback] = serviceStub[
           methodName
         ].apply(serviceStub, [req, callback]);
 
-        let cancelController, cancelSignal;
+        let cancelController: AbortController, cancelSignal;
```

Comment:
`cancelSignal` seems to be `any`?

Reply:
yes somehow the type checking does not give warning on that, added `any` type to `cancelSignal`.